LICENSE

```diff
@@ -1,4 +1,4 @@
-MIT License
+The MIT License (MIT)
```

Comment:
Not that it matters but I think we can drop `(MIT)`

Reply:
It's just the default readme template from Gatsby, I don't have preference, will revert it to what it's been.